components/script/dom/webglrenderingcontext.rs

```diff
@@ -398,10 +474,8 @@ impl WebGLRenderingContext {
 
                 let size = Size2D::new(img.width as i32, img.height as i32);
 
-                // TODO(emilio): Validate that the format argument
-                // is coherent with the image.
-                //
-                // RGB8 should be easy to support too
+                // TODO(anholt): What other incoming formats do we
```

Comment:
I think for now Servo stores all its images internally as RGBA8, so we don't need to worry about it I think.